rfc_bibtex/rfc_bibtex.py

```diff
@@ -35,8 +36,32 @@ def __init__(self, id_names=[], in_file_name=None, out_file_name=None):
         if in_file_name is not None:
             self._id_names += self._read_ids_from_file(in_file_name)
 
+    @staticmethod
```

Comment:
Is there any reason for this method to be static?

Reply:
Because it doesn't use any instance variables. It's not there to be a method on an object, it's a function that's called in a callback.

Reply:
@simsong following that logic, you could say the same about functions such as `_read_ids_from_file` :smile: I realize that this is *very* small application (no point in worrying about imposing dependencies, etc), but I don't think this should be a static. This method still does something very specific to the current class and will not be used anywhere outside of it. If your worry is in the tests where you're using it, simply make an instance of the object and call the method on it.